turnstile: add a bit of extra clarification for testing dummy keys (#14003)
cloudflare/cloudflare-docs: content/turnstile/troubleshooting/testing.md

| `2x00000000000000000000BB` | Always blocks | invisible |
| `3x00000000000000000000FF` | Forces an interactive challenge | visible |

> These dummy sitekeys will produce the `XXXX.DUMMY.TOKEN.XXXX` dummy response token.
> Production secret keys will reject this token, so for testing purposes you also need to use a dummy secret key.
> 
> 

Reviewer: ```suggestion
These dummy sitekeys will produce the `XXXX.DUMMY.TOKEN.XXXX` dummy response token.

Production secret keys will reject this token. You must also use a dummy secret key for testing purposes.

```

| Secret key | Description |
| --- | --- |
| `1x0000000000000000000000000000000AA` | Always passes |
| `2x0000000000000000000000000000000AA` | Always fails |
| `3x0000000000000000000000000000000AA` | Yields a "token already spent" error |

> Dummy secret keys will accept any response token as valid, so make sure to never use them in production.

Reviewer: ```suggestion
Dummy secret keys should never be used in production as it will accept any response token as valid. 
```
